assets/src/edit-story/components/checklist/checks/storyMissingPublisherName.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { TranslateWithMarkup, __ } from '@web-stories-wp/i18n';
+/**
+ * Internal dependencies
+ */
+import { Link, THEME_CONSTANTS } from '@web-stories-wp/design-system';
+import { useConfig } from '../../../app';
+import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
+import { PRIORITY_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+
+const StoryMissingPublisherName = () => {
+  const { generalSettingsLink, publisherName } = useConfig(
+    ({ metadata, generalSettingsLink }) => ({
+      publisherName: metadata.publisher?.name,
+      generalSettingsLink,
+    })
+  );
+  const hasPublisherName = publisherName.length > 0;
```

Comment:
This check should be a little cleverer. The title of the site could be an empty space ( something I have seen ). Making the length 1 but not render anything. In PHP there is a function [trim](https://www.php.net/manual/en/function.trim.php) removes trailing spaces, new lines etc. We should do something similar here, just in case.

Reply:
can do, when i tested it with an empty string it seemed to work so i thought some trim was happening already, but it's good to check it. will update.